tutorial/stdlib.po

```diff
@@ -377,10 +376,10 @@ msgid ""
 "applications and other tools."
 msgstr ""
 ":mod:`json` 套件對 JSON 資料交換格式的解析，提供強大的支援。\\ :mod:`csv` 模"
-"組則提供直接讀寫以逗號分隔值的檔案格式，支援一般資料庫與電子表格。\\ :mod:"
-"`xml.etree.ElementTree`\\ 、\\ :mod:`xml.dom` 與 :mod:`xml.sax` 套件則支援 "
-"XML 的處理。綜觀所有，這些模組和套件都簡化了 Python 應用程式與其他工具之間的"
-"資料交換。"
+"組則提供直接讀寫以逗號分隔值的檔案格式，通常資料庫和電子表格都有支援。\\ :"
```

Comment:
nit:

The latter sentence seems like describing `csv` module rather than the CSV format for me.


```suggestion
"組則提供直接讀寫 CSV（以逗號分隔值的檔案格式，通常資料庫和電子表格都有支援）。\\ :"
```